src/main/java/com/databasepreservation/modules/siard/out/content/SIARD1ContentExportStrategy.java

```diff
@@ -0,0 +1,424 @@
+	private final WriteStrategy writeStrategy;
+	private final SIARDArchiveContainer baseContainer;
+
+	private Set<LargeObject> LOBsToExport;
+
+	BufferedWriter currentWriter;
+	OutputStream currentStream;
+	SchemaStructure currentSchema;
+	TableStructure currentTable;
+	int currentRowIndex;
+
+	public SIARD1ContentExportStrategy(ContentPathExportStrategy contentPathStrategy, WriteStrategy writeStrategy, SIARDArchiveContainer baseContainer) {
+		this.contentPathStrategy = contentPathStrategy;
+		this.writeStrategy = writeStrategy;
+		this.baseContainer = baseContainer;
+
+		this.LOBsToExport = new HashSet<LargeObject>();
+		currentRowIndex = -1;
+	}
+
+	@Override
+	public void openTable(SchemaStructure schema, TableStructure table) throws ModuleException {
+		currentStream = writeStrategy.createOutputStream(baseContainer, contentPathStrategy.getTableXmlFilePath(schema.getIndex(), table.getIndex()));
+		currentWriter = new BufferedWriter(new OutputStreamWriter(currentStream));
+		currentSchema = schema;
+		currentTable = table;
+		currentRowIndex = 0;
+		LOBsToExport = new HashSet<LargeObject>();
+
+		try {
+			writeXmlOpenTable();
+		} catch (IOException e) {
+			throw new ModuleException("Error handling open table " + table.getId(), e);
+		}
+	}
+
+	@Override
+	public void closeTable(SchemaStructure schema, TableStructure table) throws ModuleException {
+		// finish writing the table
+		try {
+			writeXmlCloseTable();
+			currentWriter.close();
+		} catch (IOException e) {
+			throw new ModuleException("Error handling close table " + table.getId(), e);
+		}
+
+		// write lobs if they have not been written yet
+		try {
+			if( !writeStrategy.isSimultaneousWritingSupported() ){
+				for (LargeObject largeObject : LOBsToExport) {
+					writeLOB(largeObject);
+				}
+			}
+		} catch (IOException e) {
+			throw new ModuleException("Error writing LOBs", e);
+		}
+
+		// export table XSD
+		try {
+			writeXsd();
+		} catch (IOException e) {
+			throw new ModuleException("Error writing table XSD", e);
+		}
+
+		// reset variables
+		currentTable = null;
+		currentRowIndex = 0;
+	}
+
+	@Override
+	public void tableRow(Row row) throws ModuleException {
+		try {
+			currentWriter.append(TAB).append("<row>\n");
+
+			// note about columnIndex: array columnIndex starts at 0 but column columnIndex starts at 1,
+			// that is why it is incremented halfway through the loop and not at the beginning nor at the end
+			int columnIndex = 0;
+			for (Cell cell : row.getCells()) {
+				ColumnStructure column = currentTable.getColumns().get(columnIndex);
+				columnIndex++;
+				if (cell instanceof BinaryCell) {
+					writeBinaryCell(cell, column, columnIndex);
+				} else if (cell instanceof SimpleCell) {
+					writeSimpleCell(cell, column, columnIndex);
+				} else if (cell instanceof ComposedCell){
+					writeComposedCell(cell, column, columnIndex);
+				}
+				// TODO add support for composed cell types
+			}
+
+			currentWriter.append(TAB).append("</row>\n");
```

Comment:
extend stringbuilder to add prettyprint methods for adding tabs and new lines only when pretty print is enabled.